src/StarcounterClientFiles/wwwroot/sys/starcounter.html

```diff
@@ -1,2 +1,33 @@
 <link rel="import" href="palindrom-client/palindrom-client.html">
-<link rel="import" href="/sys/starcounter-include/starcounter-include.html">
\ No newline at end of file
+<link rel="import" href="/sys/starcounter-include/starcounter-include.html">
+<script>
+(function() {
+
+    function updateCurrentLinks(currentHref) {
+        const nodes = document.querySelectorAll("a[href]");
+        let lastCurrent, nextCurrent;
+        nodes.forEach((node) => {
+            if (node.hasAttribute("aria-current")) {
+                lastCurrent = node;
+            }
+
+            if (node.href === currentHref) {
+                nextCurrent = node;
+            }
+        });
+
+        if (lastCurrent && lastCurrent !== nextCurrent) {
+            lastCurrent.removeAttribute("aria-current");
+        }
+        if (nextCurrent && lastCurrent !== nextCurrent) {
+            nextCurrent.setAttribute("aria-current", "page");
+        }
+    }
+
+    window.addEventListener("stamped", (event) => {
+        if (event.target.localName === "starcounter-include") {
+            updateCurrentLinks(window.location.href);
```

Comment:
Since its a global variable I would put it as a default parameter in the function and remove it from here. 

```js
function updateCurrentLinks(currentHref = window.location.href) {
```

Reply:
I believe that the approach that I made is better for testability. It is better for a function to not depend on globals.